notebook2/notebook/notebook.py

```diff
@@ -58,11 +62,36 @@ def read_string(f):
         return f.read().strip()
 
 KUBERNETES_TIMEOUT_IN_SECONDS = float(os.environ.get('KUBERNETES_TIMEOUT_IN_SECONDS', 5.0))
-app.secret_key = read_string('/notebook-secrets/secret-key')
+
+AUTHORIZED_USERS = read_string('/notebook-secrets/authorized-users').split(',')
```

Comment:
This does not currently exist in the `notebook-secrets` secret. We need to update that secret or create a new secret and update the deployment.yaml.in appropriately.

Reply:
Got it